webrtc.html

```diff
@@ -2620,8 +2620,12 @@
           </section>
         </div>
       </section>
-      <section>
+      <section class="informative">
         <h3>Legacy Interface Extensions</h3>
+          <p>This section is non-normative; it specifies methods that can be
```

Comment:
Non-normative isn't appropriate here, sections that are non-normative should be possible to remove from the spec entirely and have no effect. I don't know of other specs that make whole sections optional, but when a single step of an algorithm is optional the wording can be "Optionally, run the following substeps" or similar. So, something like "Supporting the extensions in this section is optional" should suffice.